### ISSUE TITLE
Only 4 faces detected in video 

### ISSUE THREAD

**Issue by Chaitali-Kulkarni** [TadasBaltrusaitis/OpenFace#895]
I ran "OpenFace_2.2.0_win_x64\FaceLandmarkVidMulti.exe -f "videos\zoome.mp4" -simalign -pose -tracked". The said video contains 7 faces but only 4 faces show in the tracked video and only 4 face IDs are present in the generated csv. 

Is this a limit on th enumber of faces that can be tracked? Is there a way of increasing this limit? 

I tried it on Ubuntu and Windows and got the same limit of 4 faces.

Thanks!

**Comment by TadasBaltrusaitis** [comment]
There is a built in limit in code to do up to 4 faces. If you are able to recompile the code you can change that limit to anything you want (I should really expose that as a command line argument, but haven't gotten round to).

To change it see line 160 (`int num_faces_max = 4;`) in `FaceLandmarkVidMulti.cpp`

**Comment by Chaitali-Kulkarni** [comment]
That worked! Thank you!

**Comment by brmarkus** [comment]
Aren't actually more than the "num_faces_max" faces *detected* but only up to "num_faces_max" processed and visualized further?

**Comment by TadasBaltrusaitis** [comment]
Exactly, it restricts how many faces are processed further after original detection (mainly for performance reasons, so that face detection is not run too often).
